src/python/pants/backend/python/tasks2/pex_build_util.py

```diff
@@ -30,6 +33,10 @@ def has_python_sources(tgt):
   return isinstance(tgt, (PythonLibrary, PythonTests, PythonBinary)) and tgt.has_sources()
 
 
+def is_local_python_dist(tgt):
```

Comment:
We have a similar helper method above (`is_distribution`) and we have another call to isinstance in setup_py.py. Would it be useful to have one helper method, maybe in a util file?